src/main/java/seedu/address/model/candidate/Seniority.java

```diff
@@ -33,6 +33,10 @@ public static boolean isValidSeniority(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    public String toSearchString() {
+        return "COM" + seniority;
```

Comment:
@charmainehly char, you could possibly use Line 12's string constant.
```suggestion
        return COM_VALUE + seniority;
```